### PR TITLE
fixed GetBalance - can return float, added GetRemains

### PR DESCRIPTION
**Pull request by bvp**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -197,26 +205,49 @@ func (cli *Client) getUid() (string, error) {
...
+	}
+
+	body, _ := ioutil.ReadAll(resp.Body)
+	matches := reRemains.FindStringSubmatch(string(body))
+	if len(matches) == 0 {
+		return "", errors.New("could not find remains data")
+	}
+
+	remains := fmt.Sprintf("%s %s %s", matches[1], matches[2], matches[3])
+	if err != nil {
```
Comment (kovetskiy) [inline main.go]: There is no error here, no need for check.

File: main.go [hunk 2]
```diff
@@ -45,6 +47,12 @@ const (
 	urlChangeTariff = "https://my.yota.ru/selfcare/devices/changeOffer"
 )
 
+func (tariff Tariff) String() string {
```
Comment (kovetskiy) [inline main.go]: What is the point of this implementation? I think fmt.Sprintf("#+v", tariff) does the same, no?

**Review by kovetskiy — CHANGES_REQUESTED** [review]

**Comment by kovetskiy** [comment]
Hi! Thanks for the contribution, can you please apply a simple patch for that duplicated err checking?

File: main.go (hunk 2 above)
Comment (bvp) [reply main.go]: just for test
tests is not committed now

**Review by bvp — COMMENTED** [review]
sorry
didn't run the linters

File: main.go (hunk 1 above)
Comment (bvp) [reply main.go]: ok

Comment (bvp) [reply main.go]: how can i fix it? new commit and PR?

**Review by bvp — COMMENTED** [review]
I didn't work with the review on the github. i need help

File: main.go (hunk 1 above)
Comment (kovetskiy) [reply main.go]: @bvp  just push new commit into this branch.

**Comment by kovetskiy** [comment]
> I didn't work with the review on the github. i need help

No worries. PRs on github is just visual representation of difference between your branch and my repository master branch. So, if you want to add more changes, you do add new commit to your branch, push it to your repository and it will appear here.

File: main.go (hunk 2 above)
Comment (bvp) [reply main.go]: you're right. added because  i have a large java-background :disappointed:

**Review by bvp — COMMENTED** [review]
fixed

**Review by bvp — COMMENTED** [review]
fixed

**Comment by kovetskiy** [comment]
Thanks!

**Pull request merged**